### PR TITLE
Adding r5py.sampledata.sao_paulo

### PR DESCRIPTION
**Pull request by christophfink**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.

### CODE REVIEW

**Comment by christophfink** [comment]
@HTenkanen please comment here to confirm your role as a maintainer

**Comment by HTenkanen** [comment]
I confirm my role as a maintainer.

**Comment by christophfink** [comment]
@conda-forge/help-python, ready for review!

**Comment by christophfink** [comment]
@conda-forge/help-python, ready for review!

**Comment by christophfink** [comment]
@conda-forge/staged-recipes we've had this PR open and marked ready for review for close to 2 months now, has it gotten stuck somewhere on the way?

File: recipes/r5py.sampledata.sao_paulo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{% set name = "r5py.sampledata.sao_paulo" %}
+{% set version = "0.1.1.dev0" %}
```
Comment (ocefpaf) [inline recipes/r5py.sampledata.sao_paulo/meta.yaml:2]: This one falls under https://github.com/conda-forge/cfep/blob/main/cfep-05.md

**Comment by christophfink** [comment]
> > @conda-forge/staged-recipes we've had this PR open and marked ready for review for close to 2 months now, has it gotten stuck somewhere on the way?
> 
> conda-forge is maintained by volunteers on their spare time. There isn't a way to guarantee a recipe will be reviewed right away.

No worries, that’s what we figured, but then we pushed changes to the PR after marking is as ready for review, and were worried something had gone wrong

Going to first push a release version to pypi, then updating the recipe, thanks for the feedback!

File: recipes/r5py.sampledata.sao_paulo/meta.yaml (hunk 1 above)
Comment (christophfink) [reply recipes/r5py.sampledata.sao_paulo/meta.yaml]: We now updated it to stable version 0.1.2

**Pull request merged**